src/sidebar/store/modules/drafts.js

```diff
@@ -96,14 +97,41 @@ function createDraft(annotation, changes) {
   };
 }
 
-/** Remove all drafts. */
+/**
+ * Remove any drafts that are empty.
+ *
+ * An empty draft has no text and no reference tags.
```

Comment:
Minor nit. We usually refer to the `tags` field of an annotation as just "tags".